script5.js

```diff
@@ -0,0 +1,24 @@
+/* 5.1 */
+
+let z;
+z = 7.25;
```

Comment:
You could also do `let z = 7.25;` in one line

Reply:
Thanks I get it. I will correct.